src/admin-valid-destination-add/valid-destination-add.html

```diff
@@ -0,0 +1,89 @@
+<div class="modal" role="dialog" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h1>{{'adminValidDestinationAdd.addValidDestination' | message}}</h1>
```

Comment:
For a fixed messages you can use one time binding it will improve performance a bit.
{{:: 'adminValidDestinationAdd.addValidDestination' | message}}, instead of
{{'adminValidDestinationAdd.addValidDestination' | message}}. You can change that in a whole .html file